src/applications/caregivers/utils/helpers/schema.js

```diff
@@ -0,0 +1,27 @@
+import get from 'platform/utilities/data/get';
+import { MED_CENTERS_BY_STATE } from '../constants';
+
+export const setAddressCountry = (
+  formData,
+  _schema,
+  uiSchema,
+  _index,
+  path,
+) => {
+  const USA = { value: 'USA', label: 'United States' };
+  const addressPath = path.slice(0, -1);
+  const countryUI = uiSchema;
+  const addressFormData = get(addressPath, formData) ?? {};
+  countryUI['ui:options'].inert = true;
```

Comment:
Is setting any of these values doing anything? This looks like it returns the object at the bottom, but it is also setting some of these values here?

Reply:
Yes... so setting the `inert` to true makes the input render as read-only/a plain text value instead of an input that can be maniuplated. Since we only allow for USA, we don't to give a user any opportunity to make that value `null` and trigger a validation issue.

Reply:
So this is a variable we are setting that is being used by the forms library inputs? I think that is what is confusing to me, as in the scope of this function it does not appear to be used anywhere. As someone new to this specific code, it looked like we were arbitrarily creating a variable and then not doing anything with it.

Reply:
Correct. We're telling it to override the default behavior of the input to ensure we set the data value, limit the input to only that data value and make the input read-only to prevent any change to what we've set.

Reply:
Alright, I am understanding it now. I was confused since it looks like we are mutating the uiSchema object, which we are. That feels like it would be easy to have unintended side effects, but I do see that we do that a lot of other places. For my own knowledge, where is this logic tested?

Reply:
I'd imagine its been done from the Forms side or from the Design System side. We don't have a way to actually test the input behavior. We do check that the input exists, but cannot test the attributes and rendering itself.